templates/windows-10/default-browser.html

```diff
@@ -0,0 +1,128 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-fx-default">
+    <div id="{{ snippet_id }}-content"></div>
+
+    <script type="text/javascript">
+      //<![CDATA[
+
+      (function() {
+        var snippet = document.getElementById('{{ snippet_id }}-fx-default');
+        snippet.addEventListener('show_snippet', function () {
+          var buttons = snippet.getElementsByClassName('launch-default');
+          var snippet_content = document.getElementById('{{ snippet_id }}-content');
+
+          isDefault('appinfo', function yesDefault () {
+            sendMetric('impression-is-default');
+            snippet_content.innerHTML = '<a href="#" class="launch-default"><img class="icon" src="{{ icon_default }}" /></a><p>{{ text_default|escape }}</p>';
+
+            for (var i=0; i < buttons.length; i++) {
+              buttons[i].addEventListener('click', function (e) {
+                e.preventDefault();
```

Comment:
As alex pointed out in https://github.com/mozilla/snippets-service/pull/114/files preventing default here will break open to new tab or the right click menu on the link. For this snippet it may not be a problem -especially for the `noDefault` action but if you want to fix this copy this snippet https://github.com/mozilla/snippets-service/pull/114/files#diff-3adb3669370af06c37adc916ba7b7c3cR346

In the future and when https://github.com/mozilla/snippets-service/pull/114/files lands things will work better.